docs/android/release.md

```diff
@@ -30,11 +30,13 @@ You can download pre-built APKs for every commit on the `main` branch from the [
 ### Internal testing
 
 - Internal testers validate the release build to ensure functionality.
+- Every approved and merged PR is pushed to the internal beta channel on Google Play Store for immediate feedback. This is the only way to do real‑world testing for Android Auto/Automotive, as the debug builds will not show up in an actual vehicle.
 - Due to the app's complexity, not all features can be tested exhaustively during this phase.
 
 ### Weekly beta releases
 
-- Every week, the latest version of the `main` branch is pushed to the **open beta** channel.
+- Every Saturday at 9pm PST, the latest `main` build is pushed to the **open beta** channel.
+- Before that cutoff (Friday or Saturday), update the beta changelog to highlight new features and breaking changes.
```

Comment:
I wonder if we can't add a flag like we do for 'beta' in companion docs to identify this as a 'task'. This is blend into information about something fully automated.

Reply:
hmm well not everything is actually documented in the companion app docs, generally I try to go by the most recent commits that were merged and anything user facing I try to highlight and call out where they may see it like in settings. Most of the features are easy to write for but sometimes they are complex and 1 sentence may not be enough :) For something more important like a Breaking Change we probably def want to flag for that